SpatialGDK/Source/SpatialGDKFunctionalTests/Private/TestMaps/Spatial2WorkerSmallInterestMap.cpp

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "TestMaps/Spatial2WorkerSmallInterestMap.h"
+#include "EngineClasses/SpatialWorldSettings.h"
+#include "GameFramework/PlayerStart.h"
+#include "SpatialGDKFunctionalTests/Public/SpatialCleanupConnectionTest.h" //didn't spot this during review, but I would argue this one is "SpatialGDK" as well
+#include "SpatialGDKFunctionalTests/Public/SpatialTest1x2GridSmallInterestWorkerSettings.h"
+
+USpatial2WorkerSmallInterestMap::USpatial2WorkerSmallInterestMap()
+{
+	MapCategory = CI_FAST_SPATIAL_ONLY;
+	MapName = TEXT("Spatial2WorkerSmallInterestMap");
+}
+
+void USpatial2WorkerSmallInterestMap::CreateCustomContentForMap()
+{
+	ULevel* CurrentLevel = World->GetCurrentLevel();
+
+	// Add the tests
+	GEditor->AddActor(CurrentLevel, ASpatialCleanupConnectionTest::StaticClass(),
+					  FTransform(FVector(-50, -50, 0))); // Seems like this position is required so that the LB plays nicely?
+
+	// Quirk of the test. We need the player spawns on the same portion of the map as the test, so they are LBed together
+	AActor** PlayerStart = CurrentLevel->Actors.FindByPredicate([](AActor* Actor) {
+		return Actor->GetClass() == APlayerStart::StaticClass();
+	});
+	(*PlayerStart)->SetActorLocation(FVector(-50, -50, 100));
```

Comment:
why a 100?

Reply:
PlayerStart positions (even the original one) are slightly above ground, so anything that spawns on them isn't embedded in the ground.